posts/2020-03-31-Rust-survey-2019.md

```diff
@@ -0,0 +1,216 @@
+
+![Rust Survey Language Distribution](/images/2020-03-RustSurvey/1-Language-Breakdown.png)
+
+Here is the language distribution of the responses we received.
+
+* English: 69.6%
+* Chinese: 10.8%
+* German: 4.3%
+* French: 3.3%
+* Japanese: 3.0%
+* Polish: 1.2%
+* Portuguese: 1.2%
+* Spanish: .9%
+* Korean: .8%
+* Italian: .6%
+* Swedish: .5%
+* Vietnamese: .2%
+
+![Do You Use Rust](/images/2020-03-RustSurvey/6-Have-You-Used-Rust.png)
+
+In the 2019 survey, 82.8% of responders indicated they used Rust, 7.1% indicated they did not currently use Rust but had used it in the past, and 10.1% indicated that they had never used Rust.
+
+If we compare this to the 2018 survey (where 75% of responders indicated they used Rust, 8% indicated the did not currently use Rust but had used it in the past, and 8% indicated they had never used Rust) more responders were using Rust in 2019.
+
+## Looking Back on Rust 2018
+
+In December 2018 we released the Rust 2018 edition - Rust 1.31.0. In the 2019 survey, 92% of Rust users indicated they were using the new edition. 85% said that upgrading to the Rust 2018 edition was easy. 
+
+![Comparative improvement](/images/2020-03-RustSurvey/18-Comparative-Improvement.png)
+
+Overall, many aspects of the Rust language were percieved as "somewhat better" in the 2018 edition.
+
+## Conferences and Community
+
+We noticed some differences between English language and other language results. Within the non-English language survey subset, the majority of the issues and concerns identified are the same as those within the English language. However, one concern/trend stands out among the non-English speaking subset - a desire for Rust documentation in their native language, or the language they took the survey in. This was particularly notable within the Chinese-language group, though that is likely due to the higher representation.
+
+We are tracking the [ongoing translation efforts](https://github.com/rust-lang/book/issues?q=is%3Aopen+is%3Aissue+label%3ATranslations) with the "Translation" GitHub issue label.
+
+We received a lot of feedback on how we can improve Rust and make it feel more welcoming to more people. We can't include all of it here, so here is a summary of some of the feedback that stood out to us.
+
+People are in general asking for more learning material about Rust. In terms of expertise it's mainly beginner and intermediate level material being requested. A lot of these requests also asked for video content specifically.
+
+The common blockers that people mention to participating is that they have social anxiety, and accessibility. One of the common reasons mentioned was that some resources are hard to read for people with dyslexia.
+
+These issues are definitely ones we want to address in 2020 and beyond.
+
+## Who is using Rust and what for?
+
+![How long have you worked in Rust](/images/2020-03-RustSurvey/7-How-Long-Worked-With-Rust.png)
```

Comment:
I think if we do include pie charts, I would prefer that we have labels on each sector for the %age, given the research I've seen that suggests interpreting them correctly is otherwise quite hard :)

I would also like to see if we can perhaps avoid the close-to-duplicate colors here? There's ~3 blues that are quite similar looking at a glance, though perhaps this is related to the resolution issue I noted elsehwere.